### ISSUE TITLE
Issue with from_networkx and to_networkx

### ISSUE THREAD

**Issue by milindkesar** [rusty1s/pytorch_geometric#2018]
## 🐛 Bug
Hi, I am facing trouble understanding the citeseer dataset from planetoid which is the one obtained using 
`dataset = Planetoid(root='data/Planetoid', name='Citeseer')`
This dataset seems to have 4552 edges (undirected) and 3327 
However the paper [“Revisiting Semi-Supervised Learning with Graph Embeddings”](https://arxiv.org/abs/1603.08861) from which the dataset is taken appears to have 4732 edges and 3327 nodes.
![Screenshot from 2021-01-15 18-01-11](https://user-images.githubusercontent.com/51740623/104727896-1d976180-575c-11eb-8af0-4b17258ffc34.png)
Is there something I am missing out about the data representation? Can you please clarify this.
Thank you.

**Comment by rusty1s** [comment]
Yes, you are right. There are some duplicated edges in the `Planetoid` datasets, which we remove in a pre-processing step. The same holds true for `Cora`.
